config/main.yaml

```diff
@@ -0,0 +1,12 @@
+model:
```

Comment:
What do you think about managing the confing using [lightning CLI](https://lightning.ai/docs/pytorch/stable/cli/lightning_cli_intermediate.html)?

Reply:
Yes I did that. Nice idea, thanks. But we have to discuss how to test that and if it is actually necessary to test the training process in this case ( since it is completely handled by a third party lib )

Reply:
Reference for [testing CLI](https://github.com/Lightning-AI/pytorch-lightning/blob/1cee84ca2d8f23ed86fbd9b0a7be8b766d39ea95/tests/tests_pytorch/test_cli.py#L128C3-L155C39)

Reply:
added test - test for CLI missing :(